ospf6d/ospf6_lsa.c

```diff
@@ -551,27 +552,46 @@ void ospf6_lsa_show_dump(struct vty *vty, struct ospf6_lsa *lsa,
 {
 	uint8_t *start, *end, *current;
 	char byte[4];
+	char *header_str;
+	char adv_router[64];
```

Comment:
Can we use INET6_ADDRSTRLEN instead of 64?

Reply:
Fixed.